assets/js/admin/blocks-toggling-control.js

```diff
@@ -169,10 +171,10 @@ export const startBlocksTogglingControl = ( postType ) => {
 			} else {
 				removeNotice( 'sensei-using-blocks' );
 			}
-		} else if ( senseiBlocksCount === 0 ) {
+		} else if ( ! withSenseiBlocks ) {
```

Comment:
This check is not needed.

Reply:
Good catch! Updated here: https://github.com/Automattic/sensei/pull/4309/commits/5db99d4ca6cac6c3f039d8a219953183929ffefb